trussworks/react-uswds
# fix: Improve character count component's SR experience

# Summary

Character count includes a brief pause on input before announcing the status. 

## Related Issues or PRs

Closes #2509 


Related PR in USWDS repo: https://github.com/uswds/uswds/pull/4976

## How To Test





1. Go to http://localhost:9009/?path=/story/components-charactercount--text-input
2. Open screen reader
3. Start typing. The SR should announce number of characters left.
4. Type over limit. The message should be bold and in red. (New invalid styling.)

**The SR message for the number of characters left works in:** 

- [x] JAWS
- [x] Narrator

### Screenshots (optional)

![Screenshot 2023-08-07 at 2 39 06 PM](https://github.com/trussworks/react-uswds/assets/57681819/2a252939-b055-482f-a3a5-a91f0ed225e7)
